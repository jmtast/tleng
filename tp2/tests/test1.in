// Token desconocido: $
#tempo negra 120
#compas 3/4

const $name = value;
