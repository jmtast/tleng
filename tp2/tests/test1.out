Error:
Error de sintaxis: Token desconocido en l√≠nea 5. "$name = value;"
